docs/explanation/workload.md

```diff
@@ -0,0 +1,22 @@
+# Managing workload inside the charm
+The core jenkins agent workload requires 3 main parameters (JENKINS_URL, JENKINS_AGENT and JENKINS_SECRET) and is defined as a 2-step process:
+
+1. Download the agent binary at JENKINS_URL/jnlpJars/agent.jar and store it in the agent’s home directory
+2. Run the agent binary with the following parameters to register the node with Jenkins
+```
+/usr/bin/java -jar agent.jar \\
+-jnlpUrl "${JENKINS_URL}/computer/${JENKINS_AGENT}/slave-agent.jnlp" \\
```

Comment:
Isn't there another `.jnlp` file that we could refer to here?

Reply:
Not as I know of, since this parameter doesn't refer to a `file` but an `URL` which is hard-coded on the jenkins server. I can probably change this to `<link-to-jnlp-file>.

Reply:
I changed it to `<jnlp-path-on-jenkins-server>` to avoid any non-inclusive language.